src/main/java/se/iths/librarysystem/repository/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package se.iths.librarysystem.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import se.iths.librarysystem.entity.RoleEntity;
+
+public interface RoleRepository extends CrudRepository<RoleEntity, Long> {
+    RoleEntity findByRole(String role);
```

Comment:
I can't find where this is used. Add method call or remove.

Reply:
It's used in UserService and DefaultRolesConfig.